<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusSerialDialog</class>
 <widget class="QDialog" name="MotusSerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>80</y>
     <width>400</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>400</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>80</y>
     <width>400</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>400</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>400</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot6">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>440</y>
     <width>400</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="splitBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>16</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
